### DISCUSSION TITLE
Alternative icon for Nimble Commander

### DISCUSSION THREAD

**Discussion by vladstudio** [mikekazakov/nimble-commander#113] [Show and tell]
For my own purposes, I created an alternative icon for Nimble Commander:

![Nimble Commander](https://github.com/mikekazakov/nimble-commander/assets/914320/91fc1af9-a240-4327-9992-c93519365685)

[Nimble Commander.icns.zip](https://github.com/mikekazakov/nimble-commander/files/14075252/Nimble.Commander.icns.zip)

Preview:

<img width="757" alt="image" src="https://github.com/mikekazakov/nimble-commander/assets/914320/d90bb3f7-f54e-4a76-8740-10b1db5abffb">

**Comment by mikekazakov** [comment]
Looks nice and clean, I like it!

**Reply by vladstudio** [reply]
feel free to use!
